drivers/watchdog/wdt_mchp_xec.h

```diff
@@ -0,0 +1,47 @@
+/* wdt_xec.h - Microchip XEC watchdog driver private definitions */
+
+/*
+ * Copyright (c) 2019 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_WDT_WDT_XEC_H_
+#define ZEPHYR_DRIVERS_WDT_WDT_XEC_H_
+
+struct wdt_xec_data {
```

Comment:
Move wdt_xec_data to the .c file

Reply:
no, again there is no rule about that. I have always separated data structure into headers and code itself.